frontend/database/modules/datasets.js

```diff
@@ -642,30 +666,45 @@ const actions = {
       const { status } = err.response;
       message = `STATUS:${status} The labels of the dataset ${datasetName} with task ${datasetTask} could not be saved`;
       typeOfNotification = "error";
-      console.error(message);
+      statusCall = status;
+      if (status === 403) {
+        throw { response: TYPE_OF_FEEDBACK.NOT_ALLOWED_TO_UPDATE_LABELS };
+      }
     } finally {
-      Notification.dispatch("notify", {
-        message,
-        type: typeOfNotification,
-      });
+      statusCall === 403 ||
+        Notification.dispatch("notify", {
+          message,
+          type: typeOfNotification,
+        });
```

Comment:
1/ We need the finally here because it's fired all the time after the try/catch (except with the error is 403).
2/ A || B return A if A is truthy and return B if A is falsy